README.md

```diff
@@ -301,6 +302,43 @@ script:
       bash <(curl -s https://codecov.io/bash) -f lcov.info;
 ```
 
+#### grcov with Gitlab
+
+Here is an example `.gitlab-ci.yml` which will build your project, then collect coverage data in a format that Gitlab understands. It is assumed that you'll use an image which already has relevant tools installed, if that's not the case put the appropriate commands at the beginning of the `script` stanza.
+
+```yaml
+build:
+  variables:
+    # Set an environment variable which causes LLVM to write coverage data to the specified location. This is arbitrary, but the path passed to grcov (the first argument) must contain these files or the coverage data won't be noticed.
+    LLVM_PROFILE_FILE: "target/coverage/%p-%m.profraw"
+  script:
+    # Run all your Rust-based tests
+    - cargo test --workspace
+    # Optionally, run some other command that exercises your code to get more coverage:
+    - ./bin/integration-tests --foo bar
+    # Once a release with https://github.com/mozilla/grcov/pull/893 is available, modify this to use the multiple-outputs mechanism :)
```

Comment:
0.8.14 is now available, care to update this? :)

Reply:
It's not on `crates.io` yet, and I prefer not to point directly to github. Thank you for cutting a release, though, I'm looking forward to cutting some time off my obscene build times ;)

Reply:
0.8.15 is now available on crates.io